keps/2936-local-queue-defaulting/README.md

```diff
@@ -0,0 +1,63 @@
+# KEP-2936: LocalQueue defaulting
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+<!-- /toc -->
+
+## Summary
+
+This KEP introduces a defaulting LocalQueue mechanism for the jobs that don't have
+LocalQueue name annotation specified.
```

Comment:
```suggestion
LocalQueue name label specified.
```

The queue-name annotation has already been deprecated and will be removed in the future version.
So, could you revise the annotation to label it in all places?

Reply:
Thanks, done